bin/rucio

```diff
@@ -1003,13 +1018,28 @@ def upload(args):
     else:
         logger.debug('Skipping dataset registration')
 
+    def make_valid_did(lfn_dict):
```

Comment:
I think you do not need the `make_valid_did` part anymore (same for the rsemanager) as the dictionary describing the file will always have the `scope`, `name`, `filename` now?